src/extensions/default/InlineColorEditor/CSSProperties.json

```diff
@@ -0,0 +1,230 @@
+{
+    "align-content":               {"values": ["center", "flex-end", "flex-start", "space-around", "space-between", "stretch"]},
```

Comment:
So should I keep only those properties that are of type color and remove the rest. Also, in that case should I rename the file to ColorProperties.json?